src/olympia/versions/utils.py

```diff
@@ -135,3 +136,16 @@ def process_color_value(prop, value):
         return prop, 'rgb(%s,%s,%s)' % tuple(value)
     # strip out spaces because jquery.minicolors chokes on them
     return prop, str(value).replace(' ', '')
+
+
+def get_review_due_date(starting=None):
+    starting = starting or datetime.now()
+    # if starting falls on the weekend, move it to Monday morning
+    if starting.weekday() in (5, 6):
+        starting = starting.replace(hour=9) + timedelta(days=(7 - starting.weekday()))
+
+    due = starting + timedelta(days=REVIEWER_STANDARD_REVIEW_TIME)
+    # if due falls on a weekend, or passes over a weekend, add on 2 days
+    if due.weekday() in (5, 6) or due.weekday() < starting.weekday():
+        due += timedelta(days=2)
+    return due
```

Comment:
@wagnerand The issue said 3 business days so this code is to work around pesky weekends.  Effectively, if now is a weekend (i.e. when we're trying to set the date) then it's as if it was being set 9am Monday; and if the resulting due date lands on a weekend, or the period passes over a weekend, an extra 2 days is added on.

Reply:
Do we need to handle holidays (if so, for which country) ?

Reply:
that was why I didn't handle holidays 😬